src/ut/baseresolver_test.cpp

```diff
@@ -74,17 +68,27 @@ class BaseResolverTest : public ::testing::Test
     _baseresolver.create_srv_cache();
 
     // Create the blacklist.
-    _baseresolver.create_blacklist(30); 
+    _baseresolver.create_blacklist(30, 30);
```

Comment:
I removed the existing blacklist tests, so was free to set these values; I chose 30 and 30 so as to be consistent with the defaults.